### ISSUE TITLE
Wallet send modal no token

### ISSUE THREAD

**Issue by alaibe** [status-im/status-desktop#10119]
# Bug Report

* Disable all networks
* Click on Send button
![image](https://user-images.githubusercontent.com/491074/229440347-108f0be5-a4c3-414c-ae70-0a0c16c4c605.png)

Expected result: the list of token i can send to with correct balances is available

**Comment by stefandunca** [comment]
It seems the root of the cause is that you can unselect all networks, which makes for a strange case of no network monitored, so no tokens are selected; hence none are accessible.
![image](https://user-images.githubusercontent.com/47554641/229443312-47ba7b55-39e8-4621-8cdd-a7d57528da7e.png)
@benjthayer, this looks like the best would be to be simplified from the UX. With the current design, a quick fix is disabling the possibility of unchecking the last available enabled network. Disabling the possibility of unchecking the last network (e.g. arbitrum) will require the user to check another one to disable the "last one" (e.g. arbitrum) !?

**Comment by John-44** [comment]
> It seems the root of the cause is that you can unselect all networks, which makes for a strange case of no network monitored, so no tokens are selected; hence none are accessible. ![image](https://user-images.githubusercontent.com/47554641/229443312-47ba7b55-39e8-4621-8cdd-a7d57528da7e.png) @benjthayer, this looks like the best would be to be simplified from the UX. With the current design, a quick fix is disabling the possibility of unchecking the last available enabled network. Disabling the possibility of unchecking the last network (e.g. arbitrum) will require the user to check another one to disable the "last one" (e.g. arbitrum) !?

@stefandunca it is supposed to be impossible for a user to unselect all networks using the UI.  

1. By default, everything should be selected.  This is indicated in the UI via grey ticks

![2023-04-03 10_59_36-Window](https://user-images.githubusercontent.com/22846356/229477894-cf4cb2da-e8b8-447f-9b2c-ca5db7c00bae.png)

2. If the user click on any one item, all chains other than the item the user clicked on are deselected.  The UI looks like this

![2023-04-03 11_00_00-Window](https://user-images.githubusercontent.com/22846356/229478060-592d84bc-3579-400f-8f2e-a2703591b7ea.png)

3. If the user unselects the single selected item, it switches back to everything being selected (represented by the grey ticks)

![2023-04-03 10_59_36-Window](https://user-images.githubusercontent.com/22846356/229478220-bc5ee15b-0ea8-47dc-aa2a-8f7129464728.png)

4. Of course if the user clicks on (for example) three items, the UI should look like this

![2023-04-03 11_00_26-Window](https://user-images.githubusercontent.com/22846356/229478357-92875de1-dad3-42c1-99ce-6f3b36684110.png)

**tldr; with the design it should be impossible for the user to be in a state where no networks are selected**
